XRTK-Core/Packages/com.xrtk.core/Interfaces/Platform/IMixedRealityPlatform.cs

```diff
@@ -8,5 +8,9 @@ namespace XRTK.Interfaces.PlatformSystem
     /// </summary>
     public interface IMixedRealityPlatform : IMixedRealityDataProvider
     {
+        /// <summary>
+        /// Is this platform currently active and available?
+        /// </summary>
+        bool IsActive { get; }
```

Comment:
Not sure about the "IsActive" approach, surely as platforms are "active" they will simply register their platform with the Toolkit.  By the Platform being registered it will be active.
Shouldn't need to test if a platform is "Not" active.

Reply:
Fair enough, I was also thinking that too, but for the built in platforms there's really no way to distinguish them bc they're all always registered but not necessarily "On", make sense?

Reply:
Also, I'm not keen on the idea of services self-registering themselves. In fact I'd rather guard against that sort of thing, unless it's a trusted module.

Reply:
As discussed, can we rename this to "IsAvailable" As we are checking if the platform is available for development, not if it is active or not.